engine/src/main/java/org/destinationsol/game/BeaconHandler.java

```diff
@@ -138,7 +138,7 @@ private void updateTarget(SolGame game) {
             return;
         }
         if (farTarget == null) {
-            throw new AssertionError("Far target does not exist!");
+            throw new AssertionError("Far target is a null reference!");
```

Comment:
This is basically just a rewording - "is a null reference" is no clearer than "does not exist".